src/templates/wizardHeader/form.ejs

```diff
@@ -1,76 +1,83 @@
-{% const { properties = {} } = ctx.options %}
+{% 
+  const { panels, properties = {} } = ctx.options
```

Comment:
@shawnbot i think this code suggestion caused a build failure because panels isn't a property within ctx.options.  also, i don't think we're using `properties` anywhere in this file?

we probably want to do 

`const { panels, options = {} } = ctx` in order to get `panels = ctx.panels` and `options = ctx.options` ? 

admittedly i am not too familiar with destructuring syntax, so there's probably a better way..

Reply:
i think the first, `const { panels, options: { properties = {} } } = ctx`, will do what we're after:
- assign `panels` to `ctx.panels`
- assign `properties` to `ctx.options`

but the second (line 3) will result in `properties = {}` because `options` is not a property in `ctx`?

i'm going to commit the first suggestion anyway, but wanted to confirm my understanding 🙂